### PR TITLE
Moving go license tools from KFP repo

### PR DESCRIPTION
**Pull request by Bobgy**
CLI tools to automate go dependencies' license.

Moved from https://github.com/kubeflow/pipelines/blob/016f2a3d41bebdd57b63926dd444c09be4794b39/third_party/cli

/cc @zhenghuiwang 
/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/540)

### CODE REVIEW

**Comment by jlewi** [comment]
Please put the python files under py/kubeflow/testing

The convention we follow is that the "py" directory should be added to the python path so that all subdirectories of it are top level python packages.

We want "kubeflow" to be the top level package and then the code should be a sub package of "kubeflow.${REPO}" to avoid conflicts between code in different repositories.

**Comment by Bobgy** [comment]
Got it, I will move

**Comment by Bobgy** [comment]
Moved, does this look good?

**Comment by zhenghuiwang** [comment]
/lgtm

Only one comment.

**Comment by Bobgy** [comment]
@zhenghuiwang thanks for the review, I cannot see any comments though.

File: py/kubeflow/testing/go-license-tools/setup.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+NAME = 'kfp-license-tools'
+VERSION = '0.0.1'
+
+REQUIRES = [
+    'bs4',
+    'requests',
+    'toml',
+]
+
+setup(name=NAME,
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply py/kubeflow/testing/go-license-tools/setup.py]: Thanks for the catch!

**Comment by Bobgy** [comment]
@zhenghuiwang I fixed pylint format and package name. Can you double check? thanks

**Comment by Bobgy** [comment]
Oops, the tests are still failing, I saw the test should print pylint version, but I cannot find log for that. I think I'm using a different pylint version.

Do you have any documentation for how to fix these format errors? I cannot find which parts are wrong.

**Comment by zhenghuiwang** [comment]
It will be more helpful if the tests can print out what errors are.

Here is the `pylint` version from the testing worker image. Hope this can help you fix the lint issue.

```
$ docker run -it --entrypoint /bin/bash gcr.io/kubeflow-ci/test-worker:latest 

root@a5c629a58c0a:/# pylint --version
No config file found, using default configuration
pylint 1.8.2,
astroid 1.6.1
Python 3.5.2 (default, Jul 10 2019, 11:58:48)
[GCC 5.4.0 20160609]
```

**Comment by Bobgy** [comment]
Thanks @zhenghuiwang! I fixed the lint errors

**Comment by zhenghuiwang** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**